pkgs/development/libraries/asmjit/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  name = "asmjit";
+  version = "20221110-0c03ed2";
+
+  src = fetchFromGitHub {
+    owner = "asmjit";
+    repo = "asmjit";
+    rev = "0c03ed2f7497441ac0de232bda2e6b8cc041b2dc";
+    hash = "sha256-CfTtdgb+ZCLHwCRa+t2O4CG9rhHgqPLcfHDqLBvI9Tg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
```

Since we already use CMake I suggest adding Ninja support here as well. This will speed up the builds because it uses `NIX_BUILD_CORES` by default and does not depend on the `enableParallelBuilding` flag.
```suggestion
    cmake
    ninja
```